internal/backend/state_actions.go

```diff
@@ -147,14 +147,16 @@ func (state *State) actionAddMessagesToMailbox(
 ) (map[imap.InternalMessageID]int, error) {
 	var haveMessageIDs []MessageIDPair
```

Comment:
can remove this line; you create the variable again on line 150

Reply:
Done